tests/test_observable/test_zip.py

```diff
@@ -365,9 +365,10 @@ def test_zip_with_iterable_never_empty(self):
         n2 = []
 
         def create():
-            def mapper(x, y):
-                return x + y
-            return n1.pipe(ops.zip_with_iterable(n2, result_mapper=mapper))
+            return n1.pipe(
+                    ops.zip_with_iterable(n2),
+                    ops.map(sum),
```

Comment:
Indent not correctly (should be 4 spaces)

Reply:
this should be fixed now.